microsoft365/outlook-mcp/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ghcr.io/astral-sh/uv:python3.13-bookworm-slim
+
+# Prevents Python from writing .pyc files and buffers stdout/stderr
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+# Install system deps (if any needed) - using apt for Debian
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy uv project files first for better Docker layer caching
+COPY pyproject.toml uv.lock ./
+
+# Copy your application code
+COPY . .
+
+# Install dependencies using uv sync (faster and more reliable)
+RUN uv sync --frozen --no-dev --no-editable
```

Comment:
Does it make sense to install deps first? That way they can still be cached if the code changes